### PR TITLE
add 204. Count Primes

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 30m
Please check my code.

### CODE REVIEW

File: 204_count-primes.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class Solution:
+    def countPrimes(self, n: int) -> int:
+        """
+        time: O(n ** (3/2))
+        space: O(n)
+        """
+        judge_p = [1] * n
+        judge_p[0:2] = [0, 0]
+        for num in range(int(n ** 0.5) + 1):
+            if judge_p[num] == 1:
```
Comment (ryuji0123) [inline 204_count-primes.py:17]: if xx == 0:
    continue

File: 204_count-primes.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                for multiple in range(num ** 2, n, num):
```
Comment (ryuji0123) [inline 204_count-primes.py:18]: multiple is unclear

File: 204_count-primes.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#
+
+# @lc code=start
+class Solution:
+    def countPrimes(self, n: int) -> int:
+        """
+        time: O(n ** (3/2))
+        space: O(n)
+        """
+        judge_p = [1] * n
```
Comment (ryuji0123) [inline 204_count-primes.py:14]: Why do you use a list? You can just count prime numbers and its time complexity is the same with current one. Also, judge_p is unclear.

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

File: 204_count-primes.py (hunk 3 above)
Comment (kobakobashu) [reply 204_count-primes.py]: If I don't use a list, its time complexity is same, but time limit exceeded.  So, I used a list.

**Pull request closed without merging**